haiku-games/gloom3d/gloom3d-1.0.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="Port of the Boom3D Quake-like engine to Haiku"
+DESCRIPTION="Gloom3D is the port Boom3D engine for Haiku. \
+It uses DirectWindow for quick draw window."
+HOMEPAGE="https://github.com/HaikuArchives/Gloom3D"
+COPYRIGHT="2000 Sasha Chukov"
+LICENSE=""
+REVISION="1"
+srcGitRev="41b18a6d4038d35b16017251b226eb6c3da778d0"
+SOURCE_URI="https://github.com/HaikuArchives/Gloom3D/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256=""
+SOURCE_FILENAME="Gloom3D-$portversion-$srcGitRev"
```

Comment:
Oops. That should be `$portVersion` :)

Reply:
This should be "SOURCE_FILENAME="Gloom3D-$portversion-$srcGitRev.tar.gz"